pylint/checkers/design_analysis.py

```diff
@@ -420,14 +414,10 @@ def open(self):
             self.linter.config.exclude_too_few_public_methods
         )
 
-    def _inc_all_stmts(self, amount):
+    def _inc_all_stmts(self, amount: int) -> None:
         for i, _ in enumerate(self._stmts):
             self._stmts[i] += amount
 
-    @cached_property
-    def _ignored_argument_names(self):
```

Comment:
Unused ?

Reply:
It's an attribute lookup used once in `visit_functiondef`. I don't really see the benefit of making this a property.

Reply:
Look like there was a performance issue at some point ? Probably fixed by the migration to argparse since.